package/container/Dockerfile

```diff
@@ -43,12 +43,12 @@ RUN python3 -m pip install --upgrade --prefix=/usr pip ; \
 
 # Install decisionengine requirements (only)
 #  do as one layer to simplify merges
-RUN git clone https://github.com/HEPCloud/decisionengine.git /tmp/decisionengine.git ; \
+RUN su - adm -s /bin/bash -c "git clone https://github.com/HEPCloud/decisionengine.git /tmp/decisionengine.git "; \
     cd /tmp/decisionengine.git ; \
-    python3 setup.py bdist_wheel ; \
+    su adm -s /bin/bash -c "python3 setup.py bdist_wheel" ; \
```

Comment:
here it looks like we need
`su adm -s /bin/bash -c "cd /tmp/decisionengine.git; python3 setup.py bdist_wheel" ; \`
With this change I was able to build the docker container!

Reply:
`su` without the `-` should be leaving you in the directory you executed the command from.

Anyway, I can make that change easily enough.

Reply:
My bad, on my local Dockerfile I had `su - adm` so I had to enforce the cd to get it working.
You are right, I'm able to build the docker container with your original update.